spec/spec_helper.rb

```diff
@@ -45,6 +91,22 @@
     mocks.verify_partial_doubles = true
   end
 
+  config.before(:suite) do
+    DatabaseCleaner.strategy = :transaction
+    DatabaseCleaner.clean_with(:truncation)
+    start_es_server unless ENV['ES_SKIP']
```

Comment:
I often run single spec file tests like this:
```
% ES_SKIP=1 rspec spec/query_dsl_spec.rb
```
and it works as expected (no Elasticsearch test cluster is started/stopped).